releasenotes/notes/new-graph-decoder-and-visulization-method-e0e2a0792344a3b3.yaml

```
@@ -0,0 +1,40 @@
+---
+features:
+  - |
+    Added a graph decoder that uses clustering and
+    max_matching, defined as cluster_decoding.
+    It can be accessed similarly to matching. Change the 
+    algorithm = 'matching' to algorithm = 'clustering' to use it.
+    Draw 3d and 2d error garph method added.
+    for 2D::
+
+      code = RepetitionCode(3,1)
+      Graph = GraphDecoder(code)
+      Error_Graph = Graph.make_error_graph('1 0  00 10 00')['0']
+      Graph.draw_2d_error_graph(Error_Graph)
```

If this is drawing I would use a `jupyter-execute` directive instead so the output visualization is shown too.

```suggestion
    for 2D:
    
    .. jupyter-execute::
    
      from qiskit.ignis.verification.topological_codes.fitters import *
      
      code = RepetitionCode(3,1)
      Graph = GraphDecoder(code)
      Error_Graph = Graph.make_error_graph('1 0  00 10 00')['0']
      Graph.draw_2d_error_graph(Error_Graph)
```

---

Done